charts/connect/templates/connect-deployment.yaml

```diff
@@ -57,6 +57,7 @@ spec:
         {{- end }}
       tolerations:
       {{- toYaml .Values.connect.tolerations | nindent 8 }}
+      serviceAccountName: {{ .Values.connect.serviceAccount.name }}
```

Comment:
Should we also add this to `ClusterRoleBinding` and `RoleBinding`? Or even create a new `ClusterRoleBinding` and `RoleBinding` specific for Connect?

Checkout [this commit](https://github.com/1Password/connect-helm-charts/commit/4941a295064d8115010c052ae7bbbb4cc6456f36#diff-b860d8704ed2da97658d2bfbee0e199acb9eddcfdac59533e3c4ebacf946ce7e) for reference.